Cartfile

```diff
@@ -6,7 +6,7 @@ github "RxSwiftCommunity/RxOptional" ~> 3.1.3
 github "mozilla-mobile/telemetry-ios"
 github "jrendel/SwiftKeychainWrapper" ~> 3.0
 github "mozilla-mobile/MappaMundi" "master"
-github "mozilla/application-services" "v0.22.1"
+github "garvankeeley/application-services"      "0.27.0.1" # for xcode 10.2 compat use this fork temporarily
```

Comment:
is there already an issue to track changing this "back" to `mozilla/application-services`, or is that not necessary?

Reply:
#947